File: CalorieTrackerAPI/Common/Entities/Meal.cs
namespace Common.Entities
{
    public class Meal : BaseEntity
    {
        public Meal(string title, double calories, double carbs, double protein, double fat)
        {
            Title = title;
            Calories = calories;
            Carbs = carbs;
            Protein = protein;
            Fat = fat;
        }

        public Meal()
        {

        }

        public string Title { get; set; }
        public double Calories { get; set; }
        public double Carbs { get; set; }
        public double Protein { get; set; }
        public double Fat { get; set; }
    }
}

File: CalorieTrackerFlutter/lib/meal.dart
class Meal {
  final int id;
  final String title;
  final double calories;
  final double carbs;
  final double protein;
  final double fat;

  Meal({required this.id, required this.title, required this.calories, required this.carbs, required this.protein, required this.fat});

  Meal.fromJson( Map<String, dynamic> json)
      : title = json['title'].toString(),
        calories = double.parse(json['calories'].toString()),
        carbs = double.parse(json['carbs'].toString()),
        protein = double.parse(json['protein'].toString()),
        fat = double.parse(json['fat'].toString()),
        id = int.parse(json['id'].toString());
}